integration_tests/test_search.py

```diff
@@ -250,8 +300,8 @@ class Common(Evaluator):
     """It must be common for the results to match certain criteria.
     """
     def __init__(self, threshold=0.5, minimum=None, first_must_match=True,
-                 **kwargs):
-        """Constructor.
+                 negate=False, **kwargs):
```

Comment:
This is used by the `Uncommon` evaluator, which checks that certain results don't dominate the search results (though a few can show up).